### PR TITLE
Improve on-chain config docs

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

**Comment by Fletch153** [comment]
@samsondav there are still some "OCR2" references within the document around core node, i'm guessing these will need updating

**Review by Fletch153 — APPROVED** [review]

File: docs/Mercury.md [hunk 1]
```diff
@@ -1,162 +1,63 @@
...
-      },
-      "status": "testing"
-    }
-  },
-  "name": "Mercury v0.2 - Production Testnet Verifier (v1.0.0)",
-  "status": "testing"
+  "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
+  "chainId": 42161, // source chain id
+  "contractAddress": "0x14e044f932bb959cc2aa8dc1ba110c09224e639a", // verifier contract address
+  "configCount": 1, // the index of this config
```
Comment (kaleofduty) [inline docs/Mercury.md:21]: what does that mean, "index of this config"?

File: docs/Mercury.md [hunk 2]
```diff
@@ -1,162 +1,63 @@
...
+    "0x000....02",
+    "0x000....03",
+    "0x000....04"
+    ], // NOP signing addresses,
+    "transmitters": [
+    "0x000....11",
+    "0x000....12",
+    "0x000....13",
+    "0x000....14"
+    ], // NOP transmitter addresses
```
Comment (kaleofduty) [inline docs/Mercury.md:33]: are these filled in or do they remain dummy values?

File: docs/Mercury.md [hunk 3]
```diff
@@ -1,162 +1,63 @@
...
+      "maxDurationShouldAcceptAttestedReport": "50ms",
+      "maxDurationShouldTransmitAcceptedReport": "50ms",
+      "rMax": "25",
+      "s": [
+        4
+      ]
+    },
+  "offchainConfigVersion": 30,
+  "onchainConfig": {
+    "max": "99999999999999999999999999999",
```
Comment (kaleofduty) [inline docs/Mercury.md:56]: Is this right? I thought in DF 1.0 we use a different, higher max.

**Pull request merged**